From 20c1353fd7aa5d414b8eaa86646da23e3d29bd3a Mon Sep 17 00:00:00 2001
From: Eric Anholt <eric@anholt.net>
Date: Fri, 6 May 2016 12:59:27 -0700
Subject: [PATCH] BCM270X: Include DRM_PANEL_SIMPLE in the defconfigs.

This is going to be required for the Adafruit DPI panel support.

Signed-off-by: Eric Anholt <eric@anholt.net>
---
 arch/arm/configs/bcm2709_defconfig | 1 +
 arch/arm/configs/bcmrpi_defconfig  | 1 +
 2 files changed, 2 insertions(+)

--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -817,6 +817,7 @@ CONFIG_VIDEO_MT9V011=m
 CONFIG_DRM=m
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
 CONFIG_DRM_UDL=m
+CONFIG_DRM_PANEL_SIMPLE=m
 CONFIG_DRM_VC4=m
 CONFIG_FB=y
 CONFIG_FB_BCM2708=y
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -809,6 +809,7 @@ CONFIG_VIDEO_MT9V011=m
 CONFIG_DRM=m
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
 CONFIG_DRM_UDL=m
+CONFIG_DRM_PANEL_SIMPLE=m
 CONFIG_DRM_VC4=m
 CONFIG_FB=y
 CONFIG_FB_BCM2708=y
